.gitignore

```diff
@@ -10,15 +10,6 @@
 # Mobile Tools for Java (J2ME)
 .mtj.tmp/
 
-# Package Files #
```

Comment:
I would revert this now that you've narrowed down the jars.

Reply:
Concur